src/lang/modifyAst/addEdgeTreatment.ts

```diff
@@ -259,12 +254,12 @@ function insertParametersIntoAst(
 
 export function getPathToExtrudeForSegmentSelection(
   ast: Program,
-  selection: Selections,
+  selection: Selection,
   artifactGraph: ArtifactGraph
 ): { pathToSegmentNode: PathToNode; pathToExtrudeNode: PathToNode } | Error {
   const pathToSegmentNode = getNodePathFromSourceRange(
     ast,
-    selection.graphSelections[0]?.codeRef?.range
+    selection.codeRef?.range
   )
```

Comment:
So @max-mrgrsk I hope I'm not breaking anything there, I figured this function could be a tad more generic by taking a Selection object instead of a Selections and then gunning for the first graphSelection (so Selection) in there.